src/osd/ReplicatedBackend.cc

```diff
@@ -961,7 +961,6 @@ MOSDRepOp * ReplicatedBackend::generate_subop(
   eversion_t pg_roll_forward_to,
   hobject_t new_temp_oid,
   hobject_t discard_temp_oid,
-  const vector<pg_log_entry_t> &log_entries,
```

Comment:
could also replace this param with a `buffer::list`, which holds the encoded `log_entries`.